greenpithumb/camera.py

```diff
@@ -0,0 +1,27 @@
+class Camera(object):
+    """Wrapper for a Raspberry Pi Camera."""
+
+    def __init__(self, folder_path, local_clock, camera):
+        """Creates a new camera wrapper instance.
+
+        Args:
+            folder_path: String of the path name of the folder where pictures
+                will be stored.
+            local_clock: Local clock interface.
+            camera: Camera interface.
+        """
+        self._folder_path = folder_path
+        self._local_clock = local_clock
+        self._camera = camera
+
+    def capture(self):
+        """Captures an image from the camera."""
+        filename = self._local_clock.now().strftime('%Y-%m-%d-%H')
```

Comment:
I think we should do the full timestamp to seconds-level granularity so that we can support taking many pictures per hour or minute. Also, since we're using the filename to store the metadata about the time the picture was taken, I think we should include the UTC offset so that we can parse it back later as a timezone-aware `datetime`.

One thing to be aware of is that `:` is an illegal character on some filesystems, so we don't want to use that in our filename.

Also I don't think `picamera` will add the extension for us, so I think we need to append `'.jpg'`.